src/components/MapChart.jsx

```diff
@@ -139,12 +139,12 @@ export default class MapGenerator extends React.Component {
                 break;
             case 'usa':
                 projectionConfig.scale = USA_PROJECTION_SCALE;
-                projectionConfig.yOffset = this.props.height / USA_YOFFSET_FACTOR;
-                projectionConfig.xOffset = this.props.width / USA_XOFFSET_FACTOR;
+                projectionConfig.yOffset = 800 / USA_YOFFSET_FACTOR;
```

Comment:
Why is this hard-coded?

Reply:
This value is required to calculate the displacement of the map it won't affect the responsiveness of the chart.